extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.js

```diff
@@ -108,6 +132,76 @@ export class AmpStorySubscriptions extends AMP.BaseElement {
     }
     this.subscriptionService_.getDialog().close();
   }
+
+  /**
+   * @return {!Element}
+   * @private
+   */
+  renderSubscriptionsDialogTemplate_() {
+    return (
+      <div subscriptions-dialog subscriptions-display="NOT granted">
+        <div class="i-amphtml-story-subscriptions-dialog-banner"></div>
+        <div class="i-amphtml-story-subscriptions-dialog-content">
+          <div class="i-amphtml-story-subscriptions-publisher-text">
+            <div class="i-amphtml-story-subscriptions-price">
+              {this.element.getAttribute('price')}
+            </div>
+            <div class="i-amphtml-story-subscriptions-title">
+              {this.element.getAttribute('title')}
+            </div>
+            <div class="i-amphtml-story-subscriptions-subtitle-first">
+              {this.element.getAttribute('subtitle-first')}
+            </div>
+            <div class="i-amphtml-story-subscriptions-subtitle-second">
+              {this.element.getAttribute('subtitle-second')}
+            </div>
+          </div>
+          <div class="i-amphtml-story-subscriptions-button-container">
+            <div class="i-amphtml-story-subscriptions-publisher-button-container">
+              <div
+                class="i-amphtml-story-subscriptions-publisher-button"
+                subscriptions-action="subscribe"
+                subscriptions-display="NOT granted"
+              >
+                <img class="i-amphtml-story-subscriptions-publisher-logo"></img>
+                <div class="i-amphtml-story-subscriptions-publisher-button-text">
```

Comment:
We need to use semantic dom. This is important for a11y and document readability.
This and other elements that wrap text could be `span` elements.